documentation/docs/invoices.mdx

```diff
@@ -98,26 +98,27 @@ Content-Type: application/json
 
 ### Payment Details
 
-The Client gets the payment details of an Invoice by doing a HTTP OPTIONS of the corresponding Invoice name as URI. The Open 
-Payments Server MUST respond with either an Payment Details Response or an Error (TBD).
+The Client gets the payment details of an Invoice by doing an HTTP GET of the corresponding Invoice name as a URI, with an
+`Accept` HTTP header of `application/connection+json`. The Open Payments Server MUST respond with either an Payment Details 
```

Comment:
Elsewhere it says connection details are fetched using the `/connections` subpath after the invoice URL, is the correct/does this just need to be updated?